CHANGELOG.md

```diff
@@ -0,0 +1,4 @@
+# Changelog
+
+## 1.0.0
+* Add sample implementation
```

Comment:
**Suggestion:** Add a date and a brief description for the version 1.0.0 in the CHANGELOG.md file.
```suggestion
# Changelog

## 1.0.0 - 2022-01-01
* Add sample implementation
This version includes the initial implementation of the sample feature.
```